config/config-example.js

```diff
@@ -51,35 +51,11 @@ exports.wsdeflate = {
 }; */
 
 /**
- * ssl - support WSS, allowing you to access through HTTPS
- *  The client requires port 443, so if you use a different port here,
- *  it will need to be forwarded to 443 through iptables rules or
- *  something.
- */
-exports.ssl = null;
-
-/*
-// example:
-const fs = require('fs');
-exports.ssl = {
-	port: 443,
-	options: {
-		key: fs.readFileSync('./config/ssl/privkey.pem'),
-		cert: fs.readFileSync('./config/ssl/fullchain.pem'),
-	},
-};
-*/
-
-/*
-Main's SSL deploy script from Let's Encrypt looks like:
-	cp /etc/letsencrypt/live/sim.psim.us/privkey.pem ~user/Pokemon-Showdown/config/ssl/
-	cp /etc/letsencrypt/live/sim.psim.us/fullchain.pem ~user/Pokemon-Showdown/config/ssl/
-	chown user:user ~user/Pokemon-Showdown/config/ssl/privkey.pem
-	chown user:user ~user/Pokemon-Showdown/config/ssl/fullchain.pem
-*/
+ * TODO: allow SSL to actually be possible to use for third-party servers at
+ * some point.
```

Comment:
botched rebaseeeee

Reply:
Yea I kinda fucked up. But i was able to revert that commit thank god

Reply:
Also im curious on how you were able to review that outdated commit?

Reply:
I clicked on it in notifications before you reverted it, I think.